src/adapters/github.js

```diff
@@ -137,8 +137,9 @@ class GitHub extends Adapter {
       return cb()
     }
 
-    const username = match[1]
-    const reponame = match[2]
+    let username = ($('.commit-ref.head-ref').attr('title') || '/').match(/(.*?)\//)[1] ||
```

Comment:
So the selector to get username is the same as the one to get branch name (line 160)?